param/coercer/coercer.go

```diff
@@ -80,6 +61,7 @@ const (
 	booleanType = "boolean"
 	integerType = "integer"
 	numberType  = "number"
+	stringType  = "string"
```

Comment:
Can you keep this ordered alphabetically so that `stringType` appears below `objectType`?